scenarios/kubernetes_e2e.py

```diff
@@ -592,7 +592,7 @@ def main(args):
         set_up_kops_aws(mode.workspace, args, mode, cluster, runner_args)
     elif args.deployment == 'kops' and args.provider == 'gce':
         set_up_kops_gce(mode.workspace, args, mode, cluster, runner_args)
-    elif args.gce_ssh:
+    elif args.deployment != 'kind' and args.gce_ssh:
```

Comment:
it seems to me like this condition will always be true due to the defaulting to JENKINS_GCE_SSH_PRIVATE_KEY_FILE
```
if args.gce_ssh:
  // always true
```